spec/requests/shows_spec.rb

```diff
@@ -0,0 +1,22 @@
+require 'rails_helper'
+load "#{Rails.root}/db/seeds.rb"
+
+RSpec.describe ShowsController, type: :feature do
+  describe "GET shows#index" do
+    it 'renders index and has the header' do
+      visit '/'
+      expect(page).to have_text 'Smartflix'
+    end
+
+    context "when data is seeded" do
+      it 'includes the shows titles' do
+        visit "/"
+          expect(page).to have_text('My Little Pony: A New Generation')
+      end
+      it 'should limit to 20 shows' do
+        visit "/"
+        expect(page).not_to have_text('Monsters Inside')
```

Comment:
This test is very tightly coupled to your seed data (see above comment), is there a way that we could count the number of shows rendered on the page instead?